src/lib/chips/chip-list.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ContentChildren,
+  ElementRef,
+  ModuleWithProviders,
+  NgModule,
+  QueryList,
+  ViewEncapsulation
+} from '@angular/core';
+
+import {MdChip} from './chip';
+
+@Component(MdChipList.COMPONENT_CONFIG)
+export class MdChipList {
+
+  static COMPONENT_CONFIG: Component = {
+    moduleId: module.id,
+    selector: 'md-chip-list, [md-chip-list]',
```

Comment:
I don't think we need to do the attribute selector until we have a strong use-case for it.